environment/stats/src/cloud/benchflow/collectors/stats.go

```diff
@@ -105,6 +104,50 @@ func collectStats(container Container) {
 	}()
 }
 
+func collectNetworkStats(container Container, client docker.Client) {
+	go func() {
+		foNet, err := os.Create("/app/"+container.Name+"_network_tmp")
+		foTop, err := os.Create("/app/"+container.Name+"_top_tmp")
+		if err != nil {
+	        panic(err)
+	    }
+		interfaces, err := net.Interfaces()
+		if err != nil {
+	        panic(err)
+	    	}
+		var interfaceNames []string
+		interfaceNames = append(interfaceNames, "-t")
+		for _, each := range interfaces {
+			interfaceNames = append(interfaceNames, each.Name)
+			}
+		cmd := exec.Command("/usr/usr/local/sbin/nethogs", interfaceNames...)
+		cmd.Stdout = foNet
+		err = cmd.Start()
+	    if err != nil {
+	        panic(err)
+	    	}
+		for true {
+			select {
+			default:
+				top, err := client.TopContainer(container.Name, "")
+				if err != nil {
+		        	panic(err)
+		    	}
+				var e docker.Env
```

Comment:
Why should I comment that? I need it for resolving the top into a JSON representation